.github/workflows/continuous-integration.yml

```diff
@@ -30,6 +30,9 @@ jobs:
         with:
           php-version: 8.1
 
+      - name: Composer update
+        run: composer update
```

Comment:
Denk dat we install willen doen, anders heeft die lock niet zoveel zin toch?

Reply:
![CleanShot 2024-08-21 at 10 01 52@2x](https://github.com/user-attachments/assets/9a9c1b7e-b797-44c1-afb8-c85831ee6142)


![CleanShot 2024-08-21 at 10 02 05@2x](https://github.com/user-attachments/assets/4213915f-8420-4b18-b376-e97f8be63ed7)

Ik zit even te kijken hoe ik dit nu precies kan fixen, want de vorige keer had ik dit zelfde trucje ook gedaan en toen werkte het, ik deed toen eerste composer update en toen install en toen weer weghalen zodat alles op GitHub 'update to date' zou zijn

Reply:
Voordat ik verder ga, ga ik even kijken of ik GH Action lokaal kan gaan testen anders worden het wel heel veel commits 😅